### PR TITLE
Update some option labels for clarity

### PR DESCRIPTION
**Pull request by candela97**
~~1. Move some common options to the General > General section.~~ (Most of these changes were implemented already)
2. Rename some options for clarity. (Updated EN strings)

### CODE REVIEW

File: src/localization/en.json [hunk 1]
```diff
@@ -414,7 +414,7 @@
             "export": "Export",
             "reset": "Reset"
         },
-        "community_default_tab": "Default community home page tab",
+        "community_default_tab": "Default tab for the \"Community Hub\" button",
```
Comment (MxtOUT) [inline src/localization/en.json:417]: Not sure if the "Community Hub" button on app pages is the only link to such pages

File: src/localization/en.json [hunk 2]
```diff
@@ -502,23 +502,23 @@
...
-        "show_steamspy_info": "Show steamspy.com info",
-        "showallachievements": "Show achievement stats on \"All Games\" page",
+        "show_steamchart_info": "Show SteamCharts.com information",
+        "show_steamspy_info": "Show steamspy.com information",
+        "showallachievements": "Show achievement completion bars on \"All Games\" page",
         "showallstats": "Show playtime stats on \"All Games\" page",
-        "lowestprice_onwishlist": "Show on Wishlist",
-        "steamrepapi": "Show SteamRep status on profile pages",
-        "profile_steamid": "Show SteamID on profiles",
+        "lowestprice_onwishlist": "Show on wishlist item mouseover",
```
Comment (MxtOUT) [inline src/localization/en.json:517]:
```suggestion
        "lowestprice_onwishlist": "Show when hovering over wishlist item",
```

File: src/localization/en.json [hunk 3]
```diff
@@ -558,13 +558,13 @@
...
         "store_general_thirdparty": "Options for information from 3rd-party sites",
-        "purchase_dates": "Show apps purchase date on store pages",
-        "show_badge_progress": "Show apps badge progress on store pages",
+        "purchase_dates": "Show purchase date on store pages",
+        "show_badge_progress": "Show badge progress on store pages",
         "show_coupon": "Show applicable coupon on store pages",
-        "show_wishlist_link": "Show a Wishlist link on profiles",
-        "show_wishlist_count": "Show the number of items on Wishlist",
+        "show_wishlist_link": "Show wishlist links on profiles",
+        "show_wishlist_count": "Show the number of wishlisted items",
```
Comment (MxtOUT) [inline src/localization/en.json:567]:
```suggestion
        "show_wishlist_count": "Show the number of wishlisted items on profiles",
```
On some occasions you add the location of the feature ("on your wishlist" e.g.) and here you don't

File: src/localization/en.json (hunk 1 above)
Comment (candela97) [reply src/localization/en.json:417]: Oh, I thought this feature specifically targets the "Community Hub" button. Does it work elsewhere?

File: src/localization/en.json (hunk 3 above)
Comment (candela97) [reply src/localization/en.json:567]: This is a suboption. The parent option already mentions the location of the feature.
螢幕擷取畫面 2020-12-09 005633

File: src/localization/en.json (hunk 1 above)
Comment (MxtOUT) [reply src/localization/en.json:417]: Not that I know of any currently, but the feature is looking for any link like this, it doesn't query this specific element, so it's subject to change

File: src/localization/en.json (hunk 3 above)
Comment (MxtOUT) [reply src/localization/en.json:567]: Hm, thinking about this, wouldn't it be better to move this wishlist link section under "Show profile links to"?
The image could just be the Steam icon and the suboption could also fit in there indented just like here

**Comment by candela97** [comment]
> On some occasions you add the location of the feature ("on your wishlist" e.g.) and here you don't

@MxtOUT What do you think about creating a new section "App Page" to list features for app pages under "Store"? That way we can avoid repeating the location of these features. "Options for links to 3rd party sites" can also be put under "App Page" as a sub-section.

If that's sound good, how about another section "Wishlist"? There're only 2 options for wishlists atm, but I'm working on at least 2 more (wishlist checkboxes and one-click-remove-from-wishlist, and there's a a feature request for displaying hidden items).

File: src/localization/en.json (hunk 3 above)
Comment (candela97) [reply src/localization/en.json:567]: But the links under "Show profile links to" are for 3rd party links, I think we should keep it that way. Actually I was planning to rename "Show profile links to" to "Options for links to 3rd party sites", which matches "Options for information from 3rd party sites" in the Store section.

File: src/localization/en.json (hunk 1 above)
Comment (candela97) [reply src/localization/en.json:417]: ok, I'll revert this change for now.

Comment (MxtOUT) [reply src/localization/en.json:417]: Alright

**Comment by MxtOUT** [comment]
Your proposal sounds good!

**Pull request merged**